availability-store/src/networking.rs

```diff
@@ -89,7 +91,7 @@ pub(crate) const MAX_CHAIN_HEADS: usize = 5;
 /// Type alias for a bounded vector of leaves.
 pub type LeavesVec = ArrayVec<[Hash; MAX_CHAIN_HEADS]>;
 
-mod benefit {
+pub mod benefit {
```

Comment:
```suggestion
pub(crate) mod benefit {
```

Would this do as well?